doc/scripts/docgen.py

```diff
@@ -16,7 +16,7 @@
     opts, args = getopt.getopt(
         sys.argv[1:],
         'o:f:',
-        ['rst', 'help', 'nopdf', 'cache', 'test'])
+        ['rst', 'help', 'nopdf', 'cache', 'test', 'noerr'])
```

Comment:
@abergeron already added an additional flag `'check'` in #3885 that would be redundant with that one.